sql/structured.go

```diff
@@ -138,6 +138,18 @@ func (desc *IndexDescriptor) containsColumnID(colID ColumnID) bool {
 	return false
 }
 
+// fullColumnIDs returns the index column IDs including any implicit column IDs
+// for non-unique indexes.
+func (desc *IndexDescriptor) fullColumnIDs(tableDesc *TableDescriptor) []ColumnID {
```

Comment:
Yeah, that's a reasonable name. I'm actually fine with this as-is, but the assertion seems important.